.gitignore

```
@@ -157,4 +157,5 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-.idea/
\ No newline at end of file
+.idea/
+auth_keys_stepik.json
```

```suggestion
.idea/

```

В связке с https://github.com/zinka22/Selenium-cource/pull/39#discussion_r1639491447.